vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -0,0 +1,16 @@
+void call(Map args = [:]) {
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    String manifest = args.manifest ?: "manifests/${INPUT_MANIFEST}"
+    
+    def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
+    def buildManifest = lib.jenkins.BuildManifest.new(readYaml(file: "$WORKSPACE/builds/${inputManifest.build.getFilename()}/manifest.yml"))
```

Comment:
Can you log this path that the build manifest was retrieved from?

Reply:
Logged